File: 프로그래머스/0/120886. A로 B 만들기/A로 B 만들기.js
function ableAToB(before, after) {
    const A = before.split('').sort();
    const B = after.split('').sort();
    let check = A.every((number, index) => number === B[index]) ? 1 : 0;
    
    return check;
}

function solution(before, after) {
    let answer = 0;
    
    answer = ableAToB(before, after);
    
    return answer;
}

File: 프로그래머스/0/120886. A로 B 만들기/A로 B 만들기.py
def solution(before, after):
    answer = 0
    before_dict=dict()
    after_dict=dict()
    for ch in before:
        before_dict[ch]=before_dict.setdefault(ch,0)+1

    for ch in after:
        after_dict[ch]=after_dict.setdefault(ch,0)+1
    if before_dict==after_dict:
        return 1
    return 0